src/components/GameView/GameOverDialog.vue

```diff
@@ -21,6 +44,9 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex';
+import { mapState } from 'vuex';
```

Comment:
```suggestion
import { mapGetters, mapState } from 'vuex';
```
These imports are from the same package and can be combined.